components/aws/sagemaker/tests/unit_tests/tests/test_batch_transform.py

```diff
@@ -122,6 +124,15 @@ def test_pass_all_arguments(self):
                           'InstanceCount': None, 'VolumeKmsKeyId': ''}
     )
 
+  def test_stop_tranform_job(self):
+    batch_transform._utils = MagicMock()
+    batch_transform._utils.create_transform_job.return_value = 'test-batch-job'
+
+    try:
+      os.kill(os.getpid(), signal.SIGTERM)
```

Comment:
How does this work? We're killing the current PID?

Reply:
Didn't look much into it, but the asserts are succeeding and failing as expected and I think because we are interrupting the signal